.travis.yml

```diff
@@ -5,13 +5,21 @@ python:
 - '3.8'
 - '3.9'
 - '3.10'
+env:
+- PINT_VERSION=0.13
```

Comment:
Do we need to define this variable twice?

Reply:
It permutes the tests, so you run each Python version with each Pint version.  The alternative is to explicitly define the combinations we want to test.
https://docs.travis-ci.com/user/build-matrix/

Reply:
Ahhhhhh. gotcha! Makes sense.